docs/src/rules/no-restricted-imports.md

```diff
@@ -230,6 +232,56 @@ import { AllowedObject as DisallowedObject } from "foo";
 
 :::
 
+#### allowImportNames
+
+Inverse of importNames. Restricts all imports from a module, except specified allowed ones.
+
+```json
+"no-restricted-imports": ["error", {
+  "paths": [{
+    "name": "import-foo",
+    "allowImportNames": ["Bar"],
+    "message": "Please use Bar from /import-bar/baz/ instead."
+  }]
+}]
+```
+
+Examples of **incorrect** code for `allowImportNames` in `paths`:
+
+Disallowing all import names except 'AllowedObject'.
+
+::: incorrect { "sourceType": "module" }
+
+```js
+/*eslint no-restricted-imports: ["error", { paths: [{
+    name: "foo",
+    allowImportNames: ["AllowedObject"],
+    message: "Only 'AllowedObject' is allowed to be imported from 'foo'."
```

Comment:
Now this message will be shown as linting error message so we can change or remove it.

Reply:
How does this work please? Can I simply remove the `message` property? And why?

Reply:
for this example the linting error by ESLint will be : `'DisallowedObject' import from 'foo' is restricted only AllowedObject import(s) is/are allowed.`  which i think is similar to this message.

Reply:
Alright, thank you. So why can it be removed? We don't want to show this in the documentation?

Reply:
i mean we can remove or either change it to something relevant

Reply:
although messages are not mandatory  and we don't have to worry about it since we have shown it in option's schema in docs but we can use something like `Use allowed imports only`

Reply:
@Tanujkanti4441 Ah, this way! I'll change it.